operator/controller/src/main/java/io/apicurio/registry/operator/resource/ResourceFactory.java

```diff
@@ -64,11 +78,12 @@ public Deployment getDefaultAppDeployment(ApicurioRegistry3 primary) {
                 REGISTRY_APP_CONTAINER_NAME,
                 Configuration.getAppImage(),
                 List.of(new ContainerPortBuilder().withName("http").withProtocol("TCP").withContainerPort(8080).build()),
```

Comment:
Do we set the `8443` port in the container? The probes will succeed even if the connection can't be actually established. Do we make an actual connection in the tests?

Reply:
This is a complete miss on my part, we must add the container to the port. No real connection is made in the tests, but that's expected, we only check for the needed resources to be created and for the deployment to become ready.

Reply:
I think we should test the connection anyway. I know we've discussed what kind of functionality which types of tests should check (integration vs. operator), but in this case I'm not suggesting to check the connection content, just that it can be established, which is IMHO a deployment concern.